README.md

```diff
@@ -19,15 +22,28 @@ In your .NET application, add the Eppo.Sdk Package from Nuget.
 dotnet add package Eppo.Sdk
 ```
 
+## Client Mode
+In order to have the most up-to-date _Assignment_ and _Bandit_ configuration, the SDK frequently polls the configuration server. This is not ideal in a client deployment such as a mobile or web app. For these client applications, this SDK can be run in **Client Mode** where configuration loading is done ad-hoc by calling the `EppoClient.RefreshConfiguration` method.
```

Comment:
Our other client SDKs will automatically load, but just once.
I wonder if we just fold this into `InitClientMode()`